### PR TITLE
Enables Google Analytics measurement ID to be defined in containers.yaml

### LINKED ISSUE #1100: Enable Google Analytics measurement ID to be defined in containers.yaml
**Is your feature request related to a problem? Please describe.**
Enable Google Analytics measurement ID to be defined in containers.yaml

### PR DESCRIPTION
**Pull request by gcapiel**
### Description
Enable Google Analytics measurement ID to be defined in containers.yaml

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #1100

### CODE REVIEW

File: infra/containers.yaml [hunk 1]
```diff
@@ -147,6 +147,8 @@ Resources:
               Value: !FindInMap [AdminGroup, !Ref Environment, group]
             - Name: BASE_URL
               Value: !Sub "https://${DomainName}"
+            - Name: MEASUREMENT_ID
+              Value: !Sub "https://${MeasurementId}"
```
Comment (ndmckinley) [inline infra/containers.yaml:151]: You need to add `MeasurementId` to the parameters here at the top of the file.  Follow the example of `DomainName` maybe?

Comment (ndmckinley) [reply infra/containers.yaml:151]: You'll also need to add it to the main yaml file (along with the default, there), and pass it in - like `DomainName`.

Comment (gcapiel) [reply infra/containers.yaml:154]: @ndmckinley thanks.  Which is the main yaml file?

**Review by ndmckinley — APPROVED** [review]

**Comment by ndmckinley** [comment]
LGTM -  when you merge, can you watch the push to make sure staging remains in good shape?  I am almost certain it will but it's good to have someone's actual eyes on it.

**Pull request merged**